docs/general/ddl/replication.rst

```diff
@@ -1,73 +1,100 @@
 
-.. contents::
-    :local:
+CrateDB :ref:`allocates <gloss-shard-allocation>` every shard to a specific
+node. CrateDB will try to allocate the primary shard and replica shards to
+different nodes to maximize :ref:`durability <concept-durability>` and
+:ref:`resiliency <concept-resiliency>`.
 
-Configuration
-=============
+Replication can also improve read performance because any increase in the
+number of shards distributed across a cluster also increases the opportunities
+for CrateDB to `parallelize`_ query execution across multiple nodes.
 
-Defining the number of replicas is done using the
-:ref:`sql-create-table-number-of-replicas` property.
+.. TIP::
 
-Example::
+    The `CrateDB Admin UI`_ provides visual indicators of cluster health that
+    take replication status into account. Alternatively, you can query health
+    information for yourself directly from the :ref:`sys.health <sys-health>`
+    table. You can query detailed replication information from the
+    :ref:`sys.shards <sys-shards>` and :ref:`sys.allocations <sys-allocations>`
+    tables.
+
+You can configure the number of per-shard replicas :ref:`WITH
+<sql-create-table-with>` the :ref:`sql-create-table-number-of-replicas` table
+setting.
+
+For example::
 
     cr> create table my_table10 (
     ...   first_column integer,
     ...   second_column text
     ... ) with (number_of_replicas = 0);
     CREATE OK, 1 row affected (... sec)
 
-The ``number_of_replicas`` property also accepts an string as parameter that
-contains a ``range``.
-
-A range is a definition of minimum number of replicas to maximum number of
-replicas depending on the number of nodes in the cluster. The table below shows
-some examples.
-
-===== =========================================================================
-Range Explanation
-===== =========================================================================
-0-1   Will create no replicas if only one node. This will result in a yellow
-      cluster health.
-
-      One replica if more than one node.
------ -------------------------------------------------------------------------
-2-4   Table requires at least two replicas to be fully replicated.
-
-      Will create up to four replicas if nodes are added.
-
-      If you have less than three nodes, one or more replica shards will be
-      located on the same node as the primary shard. This will result in a
-      yellow cluster health.
------ -------------------------------------------------------------------------
-0-all Will expand the number of replicas to the available number of nodes.
-===== =========================================================================
-
-For details of the range syntax refer to :ref:`the CREATE TABLE
-documentation <sql-create-table-number-of-replicas>`.
-
-.. NOTE::
-
-    The number of replicas can be changed at any time.
+As well as configuring a single fixed number of pre-replica shards, you can use
```

Comment:
What do you mean with `pre-replica shards`? 

```suggestion
As well as configuring a single fixed number of replica shards (shard copies), you can use
```

Reply:
oops! good catch thanks. typo. I think I meant "per-shard replicas" ("per" not "pre" and mixed up the nouns)

I reworded:

> As well as being able to configure a fixed number of replicas, you can configure a range of values by using a string to specify a minimum and a maximum (dependent on the number of nodes in the cluster).